### DISCUSSION TITLE
util-linux 2.34 - 2.33....error message

### DISCUSSION THREAD

**Discussion by CGlogger** [catchthatrabbit/coreminer#32] [Miner]
Hi Rasti!
I am working with another SBC...more power as a Raspberry, but with linux.

After the installation and before the mining process should be start, i receive the error message about 2.33....

Can you please check this or should i change it to 2.33?

![grafik](https://user-images.githubusercontent.com/102905641/168680833-8b6cc641-b0ad-410a-95d9-3de41b15a0d0.png)

![grafik](https://user-images.githubusercontent.com/102905641/168680882-a5b7e33e-96f7-42f8-a911-5c1e9ebe4568.png)

Thank you!

Regards
Carsten

**Comment by CGlogger** [comment]
![image](https://user-images.githubusercontent.com/102905641/168746126-ca5ec770-101a-4205-b8c4-e0de9591027e.jpeg)


Hardware....information

**Comment by rastislavcore** [comment]
Thank you so much for the details. Please, feel free to raise an issue here https://github.com/catchthatrabbit/coreminer/issues/new/choose

**Reply by CGlogger** [reply]
...could you help me here please or not?
Or can you share it in another forum of Core?
